src/components/SwitchPlanPopup/SwitchPlanPopup.js

```diff
@@ -1,6 +1,7 @@
+/* eslint-disable prettier/prettier */
```

Comment:
Its there for some reason, because we are using Trans component from i18next package and we want to all text stay in one line and prettier trying to wrap this in multiple lines.

Reply:
so would be better to disable prettier for one particular line, not the whole file